Maquetado/scss/application.scss

```diff
@@ -1,2 +1,4 @@
 @import 'equalizer';
+@import 'titles';
 @import 'styles';
+@import url('https://fonts.googleapis.com/css?family=Roboto');
```

Comment:
font should be added right after the equalizer since it will be used in the files imported after it.